scripts/airflow3/env.sh

```diff
@@ -18,3 +18,8 @@ export AIRFLOW__CORE__DAGBAG_IMPORT_ERROR_TRACEBACK_DEPTH=10
 export AIRFLOW__CORE__DAGBAG_IMPORT_TIMEOUT=300
 # export AIRFLOW__LOGGING__LOGGING_LEVEL=DEBUG
 export AIRFLOW_REPO_DIR="$PWD/../airflow"
+
+
+# Until we fix the ExecutionMode.VIRTUALENV
+export AIRFLOW__COSMOS__ENABLE_SETUP_ASYNC_TASK=False
```

Comment:
It is probably worth to add a comment on https://github.com/astronomer/astronomer-cosmos/issues/1633 about this

Reply:
added a comment there https://github.com/astronomer/astronomer-cosmos/issues/1633#issuecomment-2812428496